libs/lb-fs/src/mount.rs

```diff
@@ -71,5 +71,6 @@ pub async fn umount() {
 
 #[cfg(target_os = "windows")]
 pub async fn umount() {
+    info!("todo");
```

Comment:
info is an unused important in the file when using windows. Stuck this in here since the other unmount functions used info in a similar place. 

Perhaps finishing this todo is a next action for me?